docs/using_dacy.getting_started.rst

```diff
@@ -113,6 +113,60 @@ To do this you can simply load the model using:
 
 Do note that this will add an additonal model to your pipeline, which will slow down the inference speed.
 
+Fine-grained NER
+########################################
+
+DaCy also features models with a more fine-grained Named Entity Recognition component, allowing the detection of 18 classes namely the following Named Entities:
+
+|              |                                                      |
+| ------------ | ---------------------------------------------------- |
+| PERSON       | People, including fictional                          |
+| NORP         | Nationalities or religious or political groups       |
+| FACILITY     | Building, airports, highways, bridges, etc.          |
+| ORGANIZATION | Companies, agencies, institutions, etc.              |
+| GPE__________| Countries, cities, states.                           |
+| LOCATION     | Non-GPE locations, mountain ranges, bodies of water  |
+| PRODUCT      | Vehicles, weapons, foods, etc. (not services)        |
+| EVENT        | Named hurricanes, battles, wars, sports events, etc. |
+| WORK OF ART  | Titles of books, songs, etc.                         |
+| LAW          | Named documents made into laws                       |
+| LANGUAGE     | Any named language                                   |
+
+As well as annotation for the following concepts:
+
+|          |                                             |
+| -------- | ------------------------------------------- |
+| DATE     | Absolute or relative dates or periods       |
+| TIME     | Times smaller than a day                    |
+| PERCENT  | Percentage (including *"%"*)                |
+| MONEY    | Monetary values, including unit             |
+| QUANTITY | Measurements, as of weight or distance      |
+| ORDINAL  | "first", "second"                           |
+| CARDINAL | Numerals that do no fall under another type |
+
+The fine-grained NER component may be utilized in an existing pipeline in the following fashion:
+
+.. code-block:: python
+
+   # load the small dacy model excluding the NER component
+   nlp = dacy.load("da_dacy_small_trf-0.1.0", exclude=["ner"])
+   # or use an empty spacy model if you only want to do NER
+   # nlp = spacy.blank("da")
+
+   # add the ner component from the state-of-the-art model
+   nlp.add_pipe("dacy/ner-small-fine-grained")
```

Comment:
This is not yet implemented in DaCy. How do we do that?

Reply:
It can be implemented as a spacy component. I will do that. If you give me the code you would use to add it to an existing pipeline I can do all the wrangling.

Reply:
@emiltj ping

Reply:
I would do it like this:

```
import spacy

!pip install https://huggingface.co/chcaa/da_dacy_small_DANSK_ner/resolve/main/da_dacy_small_DANSK_ner-any-py3-none-any.whl
nlp = spacy.load("da_core_news_sm", exclude="ner")
nlp_ner = spacy.load("da_dacy_small_DANSK_ner")
nlp.add_pipe(factory_name="transformer", source=nlp_ner)
nlp.add_pipe(factory_name="ner", source=nlp_ner)

doc = nlp("Ord som Aarhus og kl. 07:30 bliver i denne tekst annoteret")

for ent in doc.ents:
    print(ent)
    print(ent.label_)

# Aarhus
# GPE
# kl. 07:30
# TIME
```

However, I packaged and published the models with the wrong names and it seems that it is not possible to change the names of the models. So perhaps the pip install link will change.

Reply:
I trained some new models, and as a result I would now change the name of the model to the appropriate name, i.e.:

```
import spacy

!pip install pip install https://huggingface.co/chcaa/da_dacy_small_ner_fine_grained/resolve/main/da_dacy_small_ner_fine_grained-any-py3-none-any.whl
nlp = spacy.load("da_core_news_sm", exclude="ner")
nlp_ner = spacy.load("da_dacy_small_ner_fine_grained")
nlp.add_pipe(factory_name="transformer", source=nlp_ner)
nlp.add_pipe(factory_name="ner", source=nlp_ner)

doc = nlp("Ord som Aarhus og kl. 07:30 bliver i denne tekst annoteret")

for ent in doc.ents:
    print(ent)
    print(ent.label_)

# Aarhus
# GPE
# kl. 07:30
# TIME
```

Reply:
hmm but I assume this does not work using a dacy model as it will overwrite the original transformer and thus break the other components?

Reply:
This [thread](https://github.com/explosion/spaCy/discussions/6366) seems to have a solution which solved that case.